Silicon/MeteorlakePkg/Library/IgdOpRegionLib/IgdOpRegionLib.inf

```diff
@@ -0,0 +1,56 @@
+#/*++
+#
+#  Copyright (c)  1999  - 2014, Intel Corporation. All rights reserved
+#
+# This program and the accompanying materials are licensed and made available under
+# the terms and conditions of the BSD License that accompanies this distribution.
+# The full text of the license may be found at
+# http://opensource.org/licenses/bsd-license.php.
+#
+# THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+# WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
```

Comment:
The license need change to BSD plus one. Pleas check if you could use common lib Silicon\CommonSocPkg\Library\IgdOpRegionLib instead.

Reply:
patch submitted with removed duplicate lib